org.lflang.tests/src/org/lflang/tests/lsp/TestLanguageClient.java

```diff
@@ -0,0 +1,60 @@
+package org.lflang.tests.lsp;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+import org.eclipse.lsp4j.Diagnostic;
+import org.eclipse.lsp4j.DiagnosticSeverity;
+import org.eclipse.lsp4j.MessageActionItem;
+import org.eclipse.lsp4j.MessageParams;
+import org.eclipse.lsp4j.PublishDiagnosticsParams;
+import org.eclipse.lsp4j.ShowMessageRequestParams;
+import org.eclipse.lsp4j.services.LanguageClient;
+
+public class TestLanguageClient implements LanguageClient {
```

Comment:
Maybe call this `MockLanguageClient` to avoid the suggestion that this class has actual tests in it?